nitransforms/io/fsl.py

```diff
@@ -46,14 +46,14 @@ def from_ras(cls, ras, moving=None, reference=None):
 
         # Adjust for reference image offset and orientation
         refswp, refspc = _fsl_aff_adapt(reference)
-        pre = reference.affine.dot(inv(refspc).dot(inv(refswp)))
+        pre = reference.affine @ inv(refswp @ refspc)
```

Comment:
I think it makes sense to think of the matrices as `aff` and `swp @ spc`. So you are always either `aff @ inv(swp @ spc)` or `swp @ spc @ inv(aff)`.